torch/utils/data/dataset.py

```diff
@@ -194,16 +194,21 @@ def __init__(self, datasets):
     def __len__(self):
         return self.cumulative_sizes[-1]
 
-    def __getitem__(self, idx):
-        if idx < 0:
-            if -idx > len(self):
-                raise ValueError("absolute value of index should not exceed dataset length")
-            idx = len(self) + idx
+    def get_idxs(self, idx):
```

Comment:
Can you add a test to https://github.com/pytorch/pytorch/blob/13120bf67742e01d5e047b04f3b1e29e0a313b80/test/test_dataloader.py#L254 ?

Reply:
this needs a much better name too

Reply:
Unfortunately I am not too familiar with naming conventions in `PyTorch`, so I tried to keep it short.

I prefer more verbose names, e.g. `get_dataset_sample_idxs` or something similar..Anything better comes to mind?

P.S. I will add the test too ofc

Reply:
e8c095f  introduced a test for `get_idxs`

I am not so sure about f4d924e

`get_idxs` will not give an `IndexError` if the user requires an index that's out of bounds, so added a failure case for when the user actually tries to use the returned index on the data set itself. Any opinions of how to actually handle this?